microannotate/generator.py

```diff
@@ -96,10 +103,27 @@ def convert(repo, commit):
             os.path.dirname(os.path.join(repo.workdir, after_path)), exist_ok=True
         )
 
+        if remove_comments:
+            try:
+                r = requests.post(
+                    f"http://localhost:{code_analysis_port}/comment?file_name={after_path}",
+                    headers={"Content-Type": "text/plain"},
+                    data=content,
+                )
+                if r.ok:
+                    content = r.text.encode("utf-8")
+            except requests.exceptions.ConnectionError as e:
+                # The code analysis server currently doesn't respond when we pass an unsupported language.
+                logger.info(f"Error connecting to code analysis server: {e}")
```

Comment:
nit: improve logging message to have the path, the revision and so on.
We may have errors with files not encoded in utf-8.

Reply:
Fixed.